ably/ably-ui
# Fix for multiple logos rendering incorrectly

Jira Ticket Link / Motivation
----



This fixes a bug where if the logo is rendered more than once on the page, and one of the instances
if it is hidden, the other instance will not show the ably shape from the logo.

This is because the `defs` in this SVG reference ids that need to be unique ids. The browser discards the “newer”
 `linearGradients` defined in the other logo, and inherits the `hidden` style from the first.

Summary of changes
----




Fixes the bug and also adds the logo to the preview app.

How do you manually test this?
----



The changes can be tested in the browser inspector on the preview app. Duplicate the logo, then add `hidden` to one of them. The other should still look ok.

Reviewer Tasks (optional)
----




Merge/Deploy Checklist
----




Frontend Checklist
----
